docs/publication.md

```diff
@@ -0,0 +1,48 @@
+# How to publish issues on Phabricator
+
+## Build analysis output
+
+The code-review bot currently supports 6 different formats to report issues (clang-tidy, clang-format, zero coverage, coverity, infer and mozlint).
+We are in the process of standardizing toward [a single format described here](analysis_format.md).
+
+New analyzers are strongly encouraged to use that format, as it's supported directly in the bot. Other formats, or more detailed versions will need an update in the bot's code itself.
+
+
+### Taskcluster artifacts
+
+Your analyzer task needs to produce a [public Taskcluster artifact] listing all the issues found in the patch.
```

Comment:
There is no link for `public Taskcluster artifact`, maybe remove the [ ] or add the link :P

Reply:
:man_facepalming: 

In the end i did not add a link because they expire pretty quickly